packages/mco/components/disaster-recovery/application-dr-status/dr-status-table.tsx

```diff
@@ -0,0 +1,226 @@
+import * as React from 'react';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { GreenCheckCircleIcon } from '@openshift-console/dynamic-plugin-sdk';
+import { DRPlacementControlKind } from 'packages/mco/types';
+import {
+  Flex,
+  FlexItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { InProgressIcon } from '@patternfly/react-icons';
+import {
+  TableComposable,
+  Tbody,
+  Tr,
+  Td,
+  Th,
+  Thead,
+  ThProps,
+} from '@patternfly/react-table';
+import { DRPoliciesType, getDRPoliciesCount } from '../../../utils';
+import { DRPC_STATUS } from './dr-status-card';
+import './dr-status-table.scss';
+
+const reactPropFix = {
+  translate: 'yes',
+};
+
+const getCurrentStatus = (drpcList: DRPlacementControlKind[]): string =>
+  drpcList.reduce((prevStatus, drpc) => {
+    const newStatus = DRPC_STATUS[drpc?.status?.phase] || '';
+    return prevStatus
+      ? [DRPC_STATUS.Relocating, DRPC_STATUS.FailingOver].includes(newStatus)
+        ? newStatus
+        : prevStatus
+      : newStatus;
```

Comment:
```suggestion
    return [DRPC_STATUS.Relocating, DRPC_STATUS.FailingOver].includes(newStatus)
        ? newStatus
        : prevStatus || newStatus;
```
Reading multiple ternary operators is hard. Would the above suggestion be sufficient to achieve the original logic?